### ISSUE TITLE
Opsz rap order

### ISSUE THREAD

**Issue by dberlow** [TypeNetwork/Roboto-Extremo#28]
The current rap order in not in opsz numerical order.

Can we start the rap at 8 pt, run the rap of That opsz, end at the opsz 8 default
interpolate to 14 pt, run the rap of That  opsz, end at the opsz 14 default
Interpolate to 144 pt run the rap of That  opsz, end at the opsz 144 default
interpolate to 14 pt
End?

**Comment by chrissam42** [comment]
Yes. All the axes currently run in default-min-max order, do you want to change them all to min-default-max?

**Comment by dberlow** [comment]
Nope, just opsz please. The rest of the raps are fine.

On Jul 29, 2019, at 5:40 PM, Chris Lewis <notifications@github.com> wrote:

Yes. All the axes currently run in default-min-max order, do you want to change them all to min-default-max?
